src/main/resources/application.yml

```
@@ -18,11 +27,25 @@ cwa-testresult-server:
   url: http://localhost:1234
 
 keycloak:
-  auth-server-url:
+  auth-server-url: https://quicktest-iam-6155b8d7d3c6.coronawarn.app/auth/
```

Please do not expose dev env urls

```suggestion
  auth-server-url:
```